src/index.ts

```diff
@@ -269,36 +269,49 @@ class BytenodeWebpackPlugin implements WebpackPluginInstance {
   }
 }
 
-function prepare(context: string | undefined, location: string | string[], name?: string, suffix = ''): Prepared {
+function prepare(context: string | undefined, location: string | string[] | { import: string } | { import: string }[], name?: string, suffix = ''): Prepared {
   const locationArray = Array.isArray(location) ? location : [location];
 
-  const locations = locationArray.map(location => {
-    const dependency = isDependency(location);
+  const locations = locationArray
+    .map(location => {
+      if (typeof location === 'object' && location.import) {
```

Comment:
we should probably also comment why we have these conditionals in place and what they're meant to determine or substitute an isWebpack4Location(location)  and isWebpack5Location(location) to make the code self-documenting.